kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java

```diff
@@ -367,6 +468,14 @@ public void initialize() {
 
     }
 
+    private void addPlusIconToMenuItem(MenuItem menuItem) {
```

Comment:
Can you put some comments or a TODO in regards to the remove icon?

Reply:
done